pkg/build/api/validation/validation_test.go

```diff
@@ -144,7 +143,10 @@ func TestValidateBuildUpdate(t *testing.T) {
 	}
 }
 
-func TestBuildConfigValidationSuccess(t *testing.T) {
```

Comment:
@bparees it turned out that we had a test that was wrong. Would you like us to keep another `TestBuildConfigValidationSuccess` that is actually a case of valid BC?